runtime/command-start/src/main/java/io/aklivity/zilla/runtime/command/start/internal/airline/ZillaStartCommand.java

```diff
@@ -119,10 +119,8 @@ public void run()
             props.setProperty(ENGINE_WORKERS.name(), Integer.toString(workers));
         }
 
-        if (verbose)
-        {
-            props.setProperty(ENGINE_VERBOSE.name(), Boolean.toString(verbose));
-        }
+        props.setProperty(ENGINE_VERBOSE.name(), Boolean.toString(verbose));
```

Comment:
This is a change of behavior, because it will explicitly override `verbose` to false if not specified instead of picking up the value from `zilla.properties` or the `-P` value.

Please restore the `if` check and add a similar check for `exceptions` too.